lib/serverless-tools/deployer/overrides.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module ServerlessTools
+  module Deployer
+    Overrides = Struct.new(:force, :filename, keyword_init: true) do
+      def force?
+        !!force
+      end
+    end
+  end
+end
```

Comment:
I'm not really sure about this. I get the idea, these behaviours come from the cli and override whatever else is defined. But should they perhaps be merged into the config at some point? I don't really feel like the `S3Pusher` for example should care about where `force?` comes from.

I'm not even sure they are `Override`s... more just command line options. I wonder if what I really think is that the `S3Pusher` shouldn't be the thing in charge of whether or not it pushes. If something tells it to push, it should try its level best to push. Something further up should decide on whether or not to push and this would stop the passing of these overrides all the way down

Reply:
I don't think they should be merged in to the config. They're two separate entities, the configuration and the options. I think it's confusing to merge them. I have no problem renaming them to `Options` or something though better.

I'll jiggle the code about and see if they fit better in the `FunctionDeployer` class.